src/web/volta.py

```diff
@@ -79,4 +82,4 @@ def scrape_volta():
                 'contacto': tds[7].text,
             }
             writer.writerow(matriculado)
-        sleep(1)
+        sleep(1)
```

Comment:
```suggestion
        sleep(1)

```

Volver a dejar una línea al final como se recomienda siempre al final de los archivos `py`.